docs/src/resources/data.js

```diff
@@ -106,7 +106,7 @@ export const Databases = [
   {
     title: 'SQLite',
     href: 'https://www.sqlite.org/index.html',
-    imgName: 'sqllite.png',
+    imgName: 'sqlite.png',
```

Comment:
I missed renaming this file, but it's a brand so it should have the name I'm selecting.